chains/manager.go

```diff
@@ -921,47 +919,20 @@ func (m *manager) createAvalancheChain(
 	}
 
 	// create bootstrap gear
+	_, specifiedLinearizationTime := version.CortinaTimes[ctx.NetworkID]
```

Comment:
We should probably force-ably linearize non- X-chain chains as well (regardless of the networkID)